sys/kern/kern_descrip.c

```diff
@@ -921,7 +922,22 @@ kern_fcntl(struct thread *td, int fd, int cmd, intptr_t arg)
 		break;
 
 	default:
-		error = EINVAL;
+		if ((cmd & 0xffff) != F_DUP3FD) {
+			error = EINVAL;
+			break;
+		}
+		/*
+		 * Handle F_DUP3FD
+		 */
```

Comment:
No need for this comment to be multi-line.

Reply:
Fixed.